### ISSUE TITLE
Selecting text does not work when clicking on numbered lists or bullet points

### ISSUE THREAD

**Issue by Leviob** [streetwriters/notesnook#6372]
### What happened?

If you try selecting text on the web app by clicking and dragging, but you click directly on the number or the bullet point in a list, text cannot be highlighted. This often happens when trying to select text starting from the first character.

### Steps to reproduce the problem

1. Create a numbered list or unordered list
2. Try selecting text by click-and-holding on the number/bullet point and dragging to the right

### Version

3.0.13-9019490-web

### Platform/OS

Firefox

### Relevant log output

*No response*

**Comment by thecodrr** [comment]
It works if you click just a little behind the bullet or number.

**Comment by Leviob** [comment]
Yes, and I realize this is a minor bug. I just have had several times where I try selecting text and it doesn't work the first few times because my cursor happened to be exactly on the bullet or number.
